### PR TITLE
project-setup

### PR DESCRIPTION
**Pull request by darrylmorton-digicatapult**
- [x] react.js and router v6
- [x] reset.css
- [x] open sourcing
- [x] header contains logo and navigation (full width)
- [x] main contains sub navigation and page content per page-like component
- [x] navigation (with corresponding pages) and sub navigation (pages not included for now)
- [x] logo placeholder
- [x] git workflows
- [x] release notes
- [x] eslint
- [x] prettier
- [x] Dockerfile for build
- [x] check version script

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+##################################################################################################
+
+
+FROM nginx:1.18-alpine AS runtime
+
+WORKDIR /app/adcp-ui/build
+
+COPY --from=build /adcp-ui/build .
+
+CMD ["nginx", "-g", "daemon off;"]
```
Comment (dblane-digicatapult) [inline Dockerfile]: How are we passing that workdir to nginx configuration to use for hosting content?

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
Looks good to me

File: Dockerfile (hunk 1 above)
Comment (darrylmorton-digicatapult) [reply Dockerfile]: As we're not using helm for this I feel this feel should be removed.
Thoughts?

Comment (dblane-digicatapult) [reply Dockerfile]: I'd switch the docker image to using node whatever and just `npm run serve` or something.

Comment (darrylmorton-digicatapult) [reply Dockerfile]: Updated

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**